src/Utilities/functions.jl

```diff
@@ -13,35 +13,42 @@ function.
 """
 function eval_variables end
 
-function eval_variables(varval::Function, f::SVF)
```

Comment:
I'm not a fan of these acronyms. We don't need to conserve line space.

Reply:
`eval` is already quite common in Julia which is why I chose it

Reply:
I meant `SVF`, `VVF`, `SAF`, `SQF`, `VAF`, and `VQF`